front_end/html/profile_header.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
Do we need to have a separate header file? It would be preferable if we only have one so that we only have one place where we need to change things when the header needs to change.

Reply:
The header file for the profile is different because of the spacing/design. If we want to use a single file, I'll have to update the design of every other html file. So we can do that, but it will take some time.